README.md

```diff
@@ -159,6 +161,80 @@ vector.VectorObject4D.from_xyetatau(1.1, 2.2, 3.3, 4.4)
 
 and so on, for all combinations of azimuthal, longitudinal, and temporal coordinates, geometric and momentum-flavored.
 
+### SymPy vectors
+
+> Note: In order to allow SymPy simplifications to work effectively, operations on SymPy vectors are a 100% correct only if the vectors are time-like, that is, they lie in the positive time-like cone (`t**2 > x**2 + y**2 + z**2`). Operations on other vectors might output wrong results.
```

Comment:
I would phrase it more like this:

```suggestion
> **Note:** Operations on SymPy vectors are only 100% compatible with numeric vectors (Python, NumPy, and Awkward backends) if the vectors are positive time-like, that is, if `t**2 > x**2 + y**2 + z**2`. The space-like and negative time-like cases have different sign conventions.
```

because they're not exactly "wrong." For the operations in question (which ones? can we list them here in this note?), it's unclear what non-positive time-like would even mean and what the answer _should_ be. The only thing that set the choice of sign for the numeric vectors was agreement with an existing standard: ROOT. (I don't know what, if anything, set ROOT's sign convention choices.)

Having the documentation start with a disclaimer saying, "This is wrong" predisposes people to not trust it for _anything_, even though we're being specific about what the issues are. People tend to make decisions based on rough, non-granular categories (e.g. "this whole library is ^&%$*&^").

Reply:
Thanks! That sounds much better!